src/inference/src/threading/ie_istreams_executor.cpp

```diff
@@ -241,6 +323,64 @@ Parameter IStreamsExecutor::Config::GetConfig(const std::string& key) const {
     return {};
 }
 
+int IStreamsExecutor::Config::LimitHybridStreams(const int num_streams, const int num_threads) {
+    const auto core_types = custom::info::core_types();
+    const auto num_big_cores_phys = getNumberOfCPUCores(true);
+    const auto num_small_cores =
+        custom::info::default_concurrency(custom::task_arena::constraints{}.set_core_type(core_types.front()));
+    const auto base_streams_total = num_big_cores_phys + num_small_cores / 2;
+
+    // limit streams to reasonable range
+    int streams = num_streams > base_streams_total ? base_streams_total : std::max(1, num_streams);
```

Comment:
Please add warning since StreamExecutor limit and change customer setting

Reply:
Ok, I added warning info.